README.md

```diff
@@ -217,16 +223,15 @@ Distroless tracks the upstream Debian releases, using [Github actions to automat
 
 ### Debug Images
 
-Distroless images are minimal and lack shell access.  The ```:debug``` image set for each language provides a busybox shell to enter.
+Distroless images are minimal and lack shell access. The `:debug` image set for each language provides a busybox shell to enter.
```

Comment:
@loosebazooka we should probably run pre-commit on the whole repo and rebase here. Can you do that?